zhaquirks/__init__.py

```diff
@@ -119,8 +119,8 @@ class PowerConfigurationCluster(CustomCluster, PowerConfiguration):
     cluster_id = PowerConfiguration.cluster_id
     BATTERY_VOLTAGE_ATTR = 0x0020
     BATTERY_PERCENTAGE_REMAINING = 0x0021
-    MIN_VOLTS = 15  # old 2.1
-    MAX_VOLTS = 28  # old 3.2
+    MIN_VOLTS = 1.5  # old 2.1
+    MAX_VOLTS = 2.8  # old 3.2
```

Comment:
Let’s make this 3.0 since the batteries are all technically rated at 3v. They’ll get capped appropriately by the function.

Reply:
My concern with this is if the devices do not report > 2.9, then a fresh battery will never be 100%
Thoughts?

Reply:
Hrm, interesting thought... ok let’s keep it for now.